lib/DirectResourceAccessPass.cpp

```diff
@@ -106,22 +111,30 @@ bool clspv::DirectResourceAccessPass::RewriteAccessesForArg(Function *fn,
   // such a thing (where the GEP can only have zero indices).
   struct ParamInfo {
     // The base value. It is either a global variable or a resource-access
-    // builtin function. (@clspv.resource.var.* or @clspv.local.var.*)
+    // builtin function. In the global variable case it is either
+    // @clspv.resource.var.* or @clspv.local.var.*
```

Comment:
This comment doesn't make sense to me. It seems like it should either be a resource/local variable or a global variable.